Website/src/rendering/src/stories/Speakers/FeaturedSpeakers.stories.tsx

```diff
@@ -120,6 +120,26 @@ export const Default = Template.bind({});
 Default.args = {
   fields: {
     data: {
+      source: {
+        title: { value: 'Featured speakers' },
```

Comment:
Please format this new JSON with curly braces on new lines like the rest of the document.